src/App.tsx

```diff
@@ -55,9 +49,9 @@ export const App: React.FC = () => {
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && !loading && <Loader />}
```

Comment:
Seems like you made a mistake in conditional rendering and now I can see only loader instead of list of posts. I believe this is the reason of failed tests as well.

Pls double check and fix it